x/staking/keeper/msg_server.go

```diff
@@ -770,3 +801,44 @@ func (k msgServer) TransferTokenizeShareRecord(goCtx context.Context, msg *types
 
 	return &types.MsgTransferTokenizeShareRecordResponse{}, nil
 }
+
+func (k msgServer) ExemptDelegation(goCtx context.Context, msg *types.MsgExemptDelegation) (*types.MsgExemptDelegationResponse, error) {
```

Comment:
We should add a reverse operation for `ExemptDelegation`. At the moment the only way to "un-exempt" a delegation is to undelegate.

Reply:
I think you should have to undelegate.

I think un-exempt delegation by is a rare workflow

Reply:
But then, what's a user's incentive to exempt its delegation? Undelegation is expensive since there are no rewards for 3 weeks.

Reply:
The incentive to exempt delegation is that you are part of the validator team and you are contributing your ATOMs to grow your validator's business.